packages/api/src/scalars.ts

```diff
@@ -14,7 +14,7 @@ export class SanitizedString extends GraphQLScalarType {
   ) {
     super({
       // Names must match /^[_a-zA-Z][_a-zA-Z0-9]*$/ as per graphql-js
-      name: `SanitizedString_${allowedTags}_${maxLength}_${pattern}`.replace(
+      name: `SanitizedString_${allowedTags}_${maxLength}_${minLength}_${pattern}`.replace(
```

Comment:
Was this one required? I'm a bit worried this will change some API signatures.

Reply:
Yeah it is required otherwise `type`s with different `minLength` directive would fail.

Why will this change API signatures?

Reply:
I think it creates a new type. I'll pull this and run the iOS binding to verify its not an issue

Reply:
Thanks!

Reply:
ok yeah seems to have no issues